test/functional/basic/gesture-e2e-specs.js

```diff
@@ -223,6 +223,29 @@ describe('XCUITestDriver - gestures', function () {
         await doPinch();
       });
     });
+    describe('special actions', function () {
+      it('should open the control center by swiping up at the bottom', async function () {
+        await driver.elementByAccessibilityId('ControlCenterView')
+          .should.eventually.be.rejectedWith(/An element could not be located/);
+
+        const window = await driver.elementByClassName('XCUIElementTypeWindow');
+        const {width, height} = await window.getSize();
+
+        let action = new wd.TouchAction(driver);
+        action.press({
+          x: width / 2,
+          y: height - 5,
+        }).wait(500)
+        .moveTo({
+          x: width / 2,
+          y: height / 2,
```

Comment:
why not move all the way to the bottom?

Reply:
It's moving up, and there is no need to go to the top.